environment.yml

```
@@ -34,36 +31,39 @@ dependencies:
  - numpy
  - pandas
  - progressbar
- - pybind11
  - pydap>3.2.2
  - pyproj>=2.2.0
  - pyyaml
  - rasterio>=1.3.0
- - rioxarray
  - requests
+ - rioxarray
  - s3fs
  - scipy>1.10.0
+ - sentineleof>=0.9.0
```

```suggestion
 - sentineleof>=0.9.5
```
So that it resolves issue 6.10